### PR TITLE
Add patch method.

### PR DESCRIPTION
**Pull request by asgpng**

### CODE REVIEW

File: lib/proxy-client.js [hunk 1]
```diff
@@ -148,6 +148,14 @@ ProxyClient.prototype.del = function del(path) {
 };
 
 /**
+ * Make a PATCH request, returning a promise to be fulfilled with the Response
+ * or rejected with an HTTP error. See `request` for more details.
+ */
+ProxyClient.prototype.patch = function del(path) {
```
Comment (bgrainger) [inline lib/proxy-client.js]: `del` should be `patch`

Comment (asgpng) [inline lib/proxy-client.js]: Good catch. Updated.

**Comment by Schoonology** [comment]
Looks great.

**Comment by Schoonology** [comment]
Is there anything else to add (can't imagine what that would be), or should I got ahead and merge this?

**Pull request merged**